website/docs/pubsub.md

```diff
@@ -0,0 +1,217 @@
+---
+id: pubsub
+title: Consuming violations using Pubsub
+---
+
+`Feature State`: Gatekeeper version v3.13+ (alpha)
+
+> ❗ This feature is alpha, subject to change (feedback is welcome!).
+
+## Description
+
+This feature pushes audit violations to a pubsub service. Users can subscribe to pubsub service to consume violations.
+
+> To gain insights into different methods of obtaining audit violations and the respective trade-offs for each approach, please refer to [Reading Audit Results](audit.md#reading-audit-results).
+
+## Enabling Gatekeeper to export audit violations
+
+Install prerequisites such as a pubsub tool, a message broker etc.
+
+### Setting up audit with pubsub enabled
+
+In the audit deployment, set the `--enable-pub-sub` flag to `true` to publish audit violations. Additionally, `--audit-connection` and `--audit-channel` flags must be set to allow audit to publish violations. `--audit-connection` must be set to the name of the connection config, and `--audit-channel` must be set to name of the channel where violations should get published.
+
+Create a connection configMap that supplies [predefined set of configurations](pubsub-driver-walkthrough.md#how-to-use-different-providers) for a connection to get established. For instance, to establish a connection that uses Dapr to publish messages this configMap is appropriate:
```

Comment:
"that supplies predefined set of configurations" -> "that supplies a provider-specific configuration"